index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header >
+        <div class="container-main first-nav">
+                <ul class="white">
+                    
+                    <li><a href="#"><img src="./assets/mobile/facebook.png" alt="fb-logo" width="10px" height="10px"></a></li>
+                    <li><a href="#"><img src="./assets/mobile/twitter.png " alt="twit-log" width="10px" height="10px"></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Log Out</a></li>
```

Comment:
- [ ] Your design doesn't match with the given design. Please make sure it does match as much as possible. For your design, the icons, text size and font are different. 

Cloned:
![image](https://user-images.githubusercontent.com/58504655/192089081-b33451b0-930e-49af-850f-63263636379a.png)

Original:
![image](https://user-images.githubusercontent.com/58504655/192089092-37bd50cb-9cc6-4da4-b3b2-778bc0fc619b.png)

Reply:
Hello @arslanbisharat, 
Thanks for the review I will change but however, project requirement allows some adjustments **Small adjustments like text or image changes are acceptable.** That's why I build this way!